resources/hiveclients/python/hiveserver2-client.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# imports
+import sys
+import pyhs2
+from pyhs2.error import Pyhs2Exception
+
+# get the input parameters
```

Comment:
Maybe you could check that the argv array has the right number of argument (no more, no less) and raise and error (and print the usage hint) in that case.